src/best-practices/reverse-proxies.rst

```diff
@@ -303,30 +266,6 @@ with headers.
 
 For a better solution, see :ref:`api/auth/proxy`.
 
-SSL/TLS with Caddy
-------------------
-
-Caddy is https-by-default, and will automatically acquire, install, activate and,
```

Comment:
If you're dropping this section entirely, it might be good to move this text somewhere else. I wouldn't have known otherwise that Caddy is native https by default.